coderd/workspaceapps/proxy_test.go

```diff
@@ -0,0 +1,8 @@
+package workspaceapps_test
+
+// NOTE: for now, app proxying tests are still in their old locations, pending
```

Comment:
I assume this will be part of moons?

Reply:
TBD exactly, but the unit tests will live in APGL and called from both APGL and enterprise coderd with embedded and external moons/workspace proxies.